feat: add pci compliance audit functions

lamw opened the pull request:
<!-- markdownlint-disable first-line-h1 no-inline-html -->

<!--
    In order to have the best experience with our community, we recommend that you read the code of conduct and contributing guidelines before submitting a pull request.
    
    By submitting this pull request, you confirm that you have read, understood, and agreed to the project's code of conduct and contributing guidelines.

    Please use conventional commits to format the title of the pull request and the commit messages.
    For more information, please refer to https://www.conventionalcommits.org.
-->

### Summary

This change adds the following new functions to support the new VCF 5.2 PCI Compliance Audit APIs

- `Get-VCFCompliance`
- `Get-VCFComplianceConfiguration`
- `Get-VCFComplianceHistory`
- `Get-VCFComplianceTask`
- `Get-VCFComplianceStandard`
- `New-VCFCompliance`

<!--
    Please provide a clear and concise description of the pull request.
-->

### Type

<!--
    Please check the one(s) that applies to this pull request using "x".
-->

- [ ] Bugfix
- [x] Enhancement or Feature
- [ ] Code Style or Formatting
- [ ] Documentation
- [ ] Refactoring
- [ ] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

<!--
    Please check the one that applies to this pull request using "x".
    If this pull request contains a breaking change, please describe the impact and mitigation path.
-->

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### Test and Documentation

<!--
    Please check the one(s) that applies to this pull request using "x".
    For bug fixes and enhancements/features, please ensure that tests and documentation have been completed and provide details.
-->

- [x] Tests have been completed.
- [x] Documentation has been added or updated.

<!--
    Please describe the tests that have been completed and/or the documentation that has been added/updated.
-->

### Issue References

<!--
    Is this related to any GitHub issue(s)? If so, please provide the issue number(s) that are closed or resolved by this pull request.

    For bug fixes and enhancements/features, please ensure that a GitHub issue has been created and provide the issue number(s) here.

    Please use the 'Closes' keyword followed by the a hash and issue number.
    This will link the pull request to the issue(s) and automatically close them when the pull request is merged.

    Example:

    Closes #000
    Closes #001
-->

### Additional Information

<!--
    Please provide any additional information that may be helpful.
-->

Commit:
feat: Add new VCF 5.2 PCI Compliance Audit functions

Signed-off-by: William Lam <wlam@vmware.com>

tenthirtyam reviewed (commented):
mkdocs.yml needs to be updated to include the added documentation.

tenthirtyam reviewed (changes requested):
mkdocs.yml needs to be updated to include the added documentation.

tenthirtyam committed:
refactor: update

- Updates the `CHANGELOG.md`
- Updates the parameter names and descriptions for module consistency.
- Updates the documentation navigation for new functions.
- Updates the examples, as needed.
- Updates formatting for common parameters (inclusive).

Signed-off-by: Ryan Johnson <ryan.johnson@broadcom.com>

tenthirtyam commented:
@lamw - please review the updates in [ca57bb7](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation/pull/291/commits/ca57bb75eff4878cb5ef936624b07a4df0159225). 😄

tenthirtyam reviewed (approved):
LGTM - added some changes in an additional commit: 6682c6d3e99d46e60ab998a25b0fafb0e1d3fb71
